lib/misc.js

```diff
@@ -1,15 +1,15 @@
 /*
  * == BSD2 LICENSE ==
  * Copyright (c) 2014, Tidepool Project
- * 
+ *
```

Comment:
This file can be reverted as there are no non-whitespace changes. Unless, of course, if this is needed to stop the editor from continually "fixing" the file.